src/CMakeLists.txt

```
@@ -79,15 +82,32 @@ SET(root_sources
   ROOTFrameReader.cc
   ROOTLegacyReader.cc
 )
+if(ENABLE_RNTUPLE)
+  set(root_sources
+      ${root_sources}
```

```suggestion
  list(APPEND root_sources
```
Is the more idiomatic approach in CMake, I think.